lib/mumukit/bridge/runner/response_type.rb

```diff
@@ -52,10 +61,18 @@ def build_hash(response)
          test_results: [],
          status: response['exit'].to_sym}
       end
+
+      def global_status(test_status, expectation_status, response)
+        if passed_with_warnings?(test_status, expectation_status)
+          :passed_with_warnings
+        else
+          test_status
+        end
+      end
     end
 
     def self.structured_test_results?(response)
-      response['testResults'].present?
+      !response['testResults'].nil?
```

Comment:
@aguspina @flbulgarelli I'm not sure about this change.

A response type is `Unstructured` if either there are no tests or the tests has an unstructured format like `{'out' => '6 passed, 2 failed', 'exit' => 'failed'}`. In that case we won't be able to know if there are tests or not.

So I'm assuming this issue is about runners with `Structured` results, but currently the mechanism to figure out which type it is is checking if the `testResults` array is empty :sweat_smile: